apps/dg/components/case_table/case_table_adapter.js

```diff
@@ -533,12 +539,13 @@ DG.CaseTableAdapter = SC.Object.extend( (function() // closure
             }).filter( function( iCase) {
               // We filter because, at least in FireFox, we can get to a row that is one beyond the end
               // which therefore has no case
-              return !SC.none( iCase);
+              return iCase && !iCase._isProtoCase;
             }),
             select: true,
             extend: false
           };
-      tContext.applyChange( tChange);
+      if (tChange.cases.length)
```

Comment:
I believe I was just following the convention used in selectRowsInRange() immediately above, but further investigation suggests that it doesn't matter. When the context applies a `selectCases` change with `select: true`, it only selects. To deselect, one must pass `select: false`. Given that, I'm inclined to think that not applying the change is clearer than applying a no-op change which some might expect to have an effect.